### PR TITLE
Debug PoGO calculation

### PR DESCRIPTION
**Pull request by manchen07**
Hi James,

I've added the three datasets that Megan used (based on the authors and year). Could you double check whether I got the right articles? Or we could ask them for references.

I've added unit tests for these three datasets, comparing Megan's results and the package results. I think the discrepancies on Casey 1978 is caused by her way of rounding. However, for the added tests in the `test-SCD-effect-sizes.R` for PoGO, it seems that there is some discrepancy between app results and package results, and I'm not sure why...I won't have time for debugging the rest of the day, so I just send the pull request anyway. I could work more on this tomorrow if needed.

Man

### CODE REVIEW

**Review by jepusto — CHANGES_REQUESTED** [review]
1. Please rename the following files so that they do not have spaces and revise any code the calls these files:
* tests/testdata/CSES FINAL dataset.csv
* tests/testdata/CSES FINAL dataset.xlsx

File: auxiliary/prepare_POGO.R [hunk 1]
```diff
@@ -217,5 +217,44 @@ Dennis2021 <- read_excel("auxiliary/PoGO Data_to send.xlsx", sheet = 15) %>%
...
+# Byiers
+#--------------------
+Byiers2014 <- 
+  read_excel("auxiliary/CSES FINAL dataset.xlsx", sheet = 2) %>%
+  as.data.frame() %>% 
+  rename(Outcome = `Outcome (new combined outcome)`) %>% 
+  mutate(StudyID = as.character(StudyID)) %>% 
+  group_by(StudyID_CaseID) %>%
+  mutate(phase_pair_calculated = calc_phase_pairs(Condition, session = Session_number)) %>%
+  ungroup()
```
Comment (jepusto) [inline auxiliary/prepare_POGO.R:230]: Please revise so that Byiers2014 is saved as a data.frame rather than a tibble.

**Comment by jepusto** [comment]
It looks like there are two issues with the unit tests for PoGO in the web app. One was that the "by-hand" calculation of the ES did not include the argument for `aggregate = phase_pair_calculated`. I added that in the commit that I just pushed. The other issue is that there were numerical differences between the .csv file and the .xlsx file. I re-saved the csv file with a larger number of digits so that the files agree. It should work now.

**Comment by manchen07** [comment]
I've updated the data and relevant tests. Everything looks good except one error with the windows-latest CMD check. It's related to the dropdown menu of improvement direction. I'm not sure if it is a big deal since other CMD checks have passed.

**Review by jepusto — APPROVED** [review]

**Pull request merged**